src/main/java/org/example/controllers/JobRoleController.java

```diff
@@ -108,4 +111,28 @@ public Response getJobRoleById(@PathParam("id") final int id) {
                     .entity(e.getMessage()).build();
         }
     }
+    @POST
+    @RolesAllowed({UserRole.ADMIN})
+    @ApiOperation(
+            value = "Add Job Roles",
+            authorizations = @Authorization(value = HttpHeaders.AUTHORIZATION),
+            response = JobRoleRequest.class)
+    public Response createJobRole(final JobRoleRequest jobRoleRequest)
+            throws FailedToCreateException, DoesNotExistException,
```

Comment:
I cant see anywhere in the code that a DoesNotExistException is thrown in this call stack.  I think you can remove it...

Reply:
Removed